templates/taxi/driver_list.html

```diff
@@ -13,18 +15,25 @@ <h1>Driver List
         <th>Last name</th>
         <th>License number</th>
       </tr>
-    {% for driver in driver_list %}
-      <tr>
-        <td>{{ driver.id }}</td>
-        <td><a href="{{ driver.get_absolute_url }}">{{ driver.username }} {% if user == driver %} (Me){% endif %}</a></td>
-        <td>{{ driver.first_name }}</td>
-        <td>{{ driver.last_name }}</td>
-        <td>{{ driver.license_number }}</td>
-      </tr>
-    {% endfor %}
+      {% for driver in driver_list %}
+        <tr>
+          <td>{{ driver.id }}</td>
+          <td><a href="{{ driver.get_absolute_url }}">{{ driver.username }} {% if user == driver %} (Me){% endif %}</a>
```

Comment:
Ensure that the `Driver` model has a `get_absolute_url` method defined. If not, replace `{{ driver.get_absolute_url }}` with the appropriate URL tag, e.g., `{% url 'taxi:driver-detail' pk=driver.id %}`.